lib/themes/dosomething/paraneue_dosomething/js/campaign/helpers.js

```diff
@@ -1,4 +1,31 @@
 /**
+ * Checks if the given campaign is still open
+ * @param {object|null} campaign
+ * @return {bool}
+ */
+export function checkIfCampaignIsOpen(campaign = null) {
```

Comment:
Maybe rename to `isCampaignOpen`? sounds better when doing a boolean check... "is campaign open? true"